.gitignore

```diff
@@ -1,4 +1,5 @@
 node_modules/
 build/
+test-coverage/
```

Comment:
Can we make a folder called `reports` and add a folder called `jest` for all the unit test coverage details ?

This would make it easier to have a central for all types of reports like automation etc.